dockerfile/anaconda-iso-creator/lorax-build-webui

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Build a Web UI boot.iso by lorax. The boot.iso will be stored in the `/images/` directory.
+# We have to build the RPMs files of Anaconda first and then add them as volume
+# mount to /anaconda-rpms to the container (could be RO mount).
+#
+# Compared to the normal boot.iso we need to patch the templates but other than
+# that it's mostly the same.
+#
+# To run this please do:
+#
+#   sudo make -f ./Makefile.am container-rpms-scratch
+#   sudo podman run -i --rm --privileged --tmpfs /var/tmp:rw,mode=1777 -v `pwd`/result/build/01-rpm-build:/anaconda-rpms:ro -v `pwd`/output-dir:/images:z --entrypoint /lorax-build-webui quay.io/rhinstaller/anaconda-iso-creator:master
+#
+#
+# Input directory:
+# /anaconda-rpms/ (Anaconda RPM files for the build)
+#
+# Output directory:
+# /images (Where the boot.iso will be stored)
+#
+
+set -eux
+
+# pre-create loop devices manually. In the container you can't use losetup for that.
+mknod -m 0660 /dev/loop0 b 7 0  2> /dev/null || true
+mknod -m 0660 /dev/loop1 b 7 1  2> /dev/null || true
+
+INPUT_RPMS=/anaconda-rpms/
+REPO_DIR=/tmp/anaconda-rpms/
+
+# create repo from provided Anaconda RPMs
+mkdir -p $REPO_DIR
+cp -a $INPUT_RPMS/* $REPO_DIR || echo "RPM files can't be copied!"  # We could just do the build with official repositories only
+createrepo_c $REPO_DIR
+
+cp -r /usr/share/lorax/templates.d/ /lorax/
+patch -p2 -i /adjust-templates-for-webui.patch
+
+# build boot.iso with our rpms
+. /etc/os-release
+# The download.fedoraproject.org automatic redirector often selects download-ib01.f.o. for GitHub's cloud, which is too unreliable; use a mirror
+# The --volid argument can cause different network interface naming: https://github.com/rhinstaller/kickstart-tests/issues/448
+lorax -p Fedora -v "$VERSION_ID" -r "$VERSION_ID" \
+      --volid Fedora-S-dvd-x86_64-rawh \
+      --sharedir ./templates.d/99-generic/ \
+      -s http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/ \
+      -s https://fedorapeople.org/groups/anaconda/webui_new_payload/repo/ \
```

Comment:
In this repo, there is only one huge package. How do I know what it is and how to make it? Or will this change so soon that it's meaningless to document?

Reply:
Yes, that is taken from @M4rtinK tarball. It's test payload wrapped into RPM so it could be installed by Lorax.

Ideally we want this file minimal and don't care about it. We need to drop this in the future anyway.

Reply:
Its not just the payload tarball - it also has an alternative `interactive-defaults.ks` file, that actually makes the boot.iso use the tar payload without user interaction.

This is because we still don't have a config file support for setting installation payload.

Alternatively, when modularization is done, we could also use RPM + closest mirror, but that unlike the current image will not be network independent, which could be tricky as we don't have any means to configure networking on the image.